docs/Install.md

```diff
@@ -20,7 +20,7 @@ Pelorus gets installed via helm charts. The first deploys the operators on which
 
 ```shell
 # clone the repo (you can use a different release or clone from master if you wish)
-git clone --depth 1 --branch v1.6.0 https://github.com/konveyor/pelorus
+git clone --depth 1 --branch v1.7.1 https://github.com/konveyor/pelorus
```

Comment:
any chance we could add this github action to associate the latest release w/ the tag "latest" so we could execute
```
git clone --depth 1 --branch vlatest https://github.com/konveyor/pelorus
```

Reply:
Created #609